go/pkg/constants/constants.go

```diff
@@ -29,24 +29,27 @@ var StablecoinDetails = map[int64]map[Stablecoin]StablecoinData{
 }
 
 type ConfigData struct {
-	VaultAddress  string
-	VaultDecimals int64
-	SDai          string
-	LiquidityPool string
+	VaultAddress   string
+	VaultDecimals  int64
+	AdapterAddress string
```

Comment:
why? in the smart contract, NovaVault can have several adapters: one for USDC, one for USDT, etc.

Reply:
`AdapterAddress` should be removed here